src/opcodes.rs

```diff
@@ -88,99 +89,102 @@ impl Opcode {
             "revert" => REVERT,
             "invalid" => INVALID,
             "selfdestruct" => SELFDESTRUCT,
-            &_ => panic!("Unknown opcode: {}", name),
+            &_ => JUMPLABEL,
```

Comment:
treating anything else as a jump label feels wrong to me. we should probably parse it out correctly. maybe use the JumpLabelMap for that

Reply:
Agreed - this was more of a hotfix since otherwise it would error out. Will check out JumpLabelMap